### PR TITLE
Added 'MetaTags' on all public pages in Softuni Judge.

### PR DESCRIPTION
**Pull request by deyordanov**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/1439

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/profile/ProfilePage.tsx [hunk 1]
```diff
@@ -137,21 +148,19 @@ const ProfilePage = () => {
...
+                        </div>
+                    )}
+        </>
     );
 };
 
-export default setLayout(withTitle(ProfilePage, (params) => !isEmpty(params.username)
-    ? `${params.username}'s profile`
-    : 'My Profile'));
+export default ProfilePage;
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/profile/ProfilePage.tsx:166]: Isn't `setLayout` needed?

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx [hunk 2]
```diff
@@ -204,4 +210,4 @@ const ContestDetailsPage = () => {
     );
 };
 
-export default setLayout(withTitle(ContestDetailsPage, (params) => `Contest #${params.contestId}`));
+export default ContestDetailsPage;
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx:213]: Isn't `setLayout` needed?

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/profile/ProfilePage.tsx (hunk 1 above)
Comment (deyordanov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/profile/ProfilePage.tsx]: Done.

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx (hunk 2 above)
Comment (deyordanov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-details/ContestDetailsPage.tsx]: Done.

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**